fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -893,7 +893,7 @@ protected void checkIfTenantNameAndTenantKeyExists(Db2Adapter adapter, String te
                 final String sql =
                         "SELECT t.tenant_status FROM fhir_admin.tenants t WHERE t.tenant_name = ? "
                                 + "AND EXISTS (SELECT 1 FROM fhir_admin.tenant_keys tk WHERE tk.mt_id = t.mt_id "
-                                + "AND tk.tenant_hash = sysibm.hash(tk.tenant_salt || ?, 2));";
+                                + "AND tk.tenant_hash = sysibm.hash(tk.tenant_salt || ?, 2))";
```

Comment:
DB2 SQL Error: SQLCODE=-104, SQLSTATE=42601, SQLERRMC=;;enant_salt || ?, 2));END-OF-STATEMENT
Possibly double end of statement problem? maybe only occurs on Windows versus Mac?

Reply:
wouldn't our windows e2e test have caught this?  if not, why not?

Reply:
I thought so too. 

I did send a new cli jar to Frances and it worked.